example/lib/mail_page.dart

```diff
@@ -0,0 +1,200 @@
+import 'package:empty_placeholder_view/emptyplaceholderview.dart';
+import 'package:example/mail_tile.dart';
+import 'package:example/values/vectors.dart';
+import 'package:flutter/material.dart';
+
+class MailPage extends StatefulWidget {
+  const MailPage({Key? key}) : super(key: key);
+
+  @override
+  _MailPageState createState() => _MailPageState();
+}
+
+class _MailPageState extends State<MailPage> {
+  List<Mail> mails = [];
+  ApiState apiState = ApiState.loading;
```

Comment:
Let's rename the API state to some other state as it is not only intended for API calls it can be used for general purposes as well.